config/mkdocs-common.yml

```
@@ -58,6 +58,10 @@ extra:
       link: /nl/
       lang: nl
       icon: https://raw.githubusercontent.com/twitter/twemoji/master/assets/svg/1f1f3-1f1f1.svg
+    - name: 繁体字
```

We need to double-check this says "Chinese Traditional" in traditional Chinese, because I found this with a random internet search.

---

Google Translate shows both the one in the PR and the following translation as options, but suggests this one may be more correct, so let's get in touch with the people working on this translation to confirm:

```suggestion
    - name: 繁體中文
```

---

- name: 正體中文

---

```suggestion
    - name: 正體中文
```